CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 
   Fixes issues with govuk-fronted javascript clashing with govuk-template/govuk-element and potentially other third-party javascript libraries using `js-` class names as hooks.
 
+  if you have any custom JavaScript relying on any GOV.UK Frontend components having class name prefixed with `.js-`(eg. `.js-header-toggle`) class name then you will need to find and add `.govuk-js-` to it.
```

Comment:
Could people confuse this advice and add `.govuk-js-` to `.js-enabled` sass?

I wonder since we have only two classes we just make it explicit on how to update specifically those classes...

Reply:
Thats a really good point. We are only updating two css classes. I've pushed up another commit that should address this problem and also add instructions for non-nunjuck usage.